packages/eslint-config-next/package.json

```
@@ -8,6 +8,7 @@
     "url": "vercel/next.js",
     "directory": "packages/eslint-config-next"
   },
+  "homepage": "https://nextjs.org/docs/pages/building-your-application/configuring/eslint#eslint-config",
```

```suggestion
  "homepage": "https://nextjs.org/docs/app/building-your-application/configuring/eslint#eslint-config",
```

Should we default to app instead of pages? cc @delbaoliveira

---

Yes, we should.